go/kbfs/libkbfs/keybase_daemon_rpc.go

```diff
@@ -465,6 +465,7 @@ func newNotifyServiceHandler(config Config, log logger.Logger) keybase1.NotifySe
 func (k *KeybaseDaemonRPC) FavoritesChanged(ctx context.Context,
 	uid keybase1.UID) error {
 	k.log.Debug("Received FavoritesChanged RPC.")
-	k.config.KBFSOps().RefreshCachedFavorites(ctx)
+	go k.config.KBFSOps().RefreshCachedFavorites(context.Background(),
+		FavoritesRefreshModeBlocking)
```

Comment:
I still don't understand why this requires "blocking".

Reply:
I think "blocking" was probably a bad name for this. I've renamed it to `Background` and clarified in a comment.

Reply:
I think `Background` is a much worse name, since the call isn't actually made in the background.

I wasn't concerned with the name "blocking", which seems descriptive of what's going to happen.  I was confused about why blocking was needed at all, which you explained in the other comment thread.